### PR TITLE
First batch of references of my dissertation

### PR DESCRIPTION
**Pull request by jonaskuResearch**
This is the first batch of references. I tried not to include all references to keep it manageable.
Once this pull request is approved, I will include the other references.

### CODE REVIEW

File: author.bib [hunk 1]
```diff
@@ -644,30 +699,37 @@ @string{ Misailovic_S
...
 @string{ Moreno_A             = "Moreno, A." }
 @string{ Morgan_N             = "Morgan, Nelson"}
 @string{ Moses_ME             = "Moses, Melanie E." }
+@string{ Moslinger_C          = "Moslinger, Christoph" }
 @string{ Mostaghim_S          = "Mostaghim, Sanaz" }
 @string{ Mouret_JB            = "Mouret, Jean-Baptiste" }
 @string{ Mukherjee_A          = "Mukherjee, Apratim" }
 @string{ Muller_J             = "Müller, Jörg" }
 @string{ Muller_KR            = "Müller, Klaus-Robert"}
+@string{ MunozMartinez_V      = "Muñoz-Martinez, Victor" }
```
Comment (dagarzonr) [inline author.bib:725]:
```suggestion
@string{ MunozMartinez_VF      = "Muñoz Martínez, Víctor Fernando" }
```

Comment (dagarzonr) [reply author.bib:725]: I checked his papers and it is a bit inconsistent with the author name. I think we can keep them all to be comprehensive. Please double verify.

File: author.bib [hunk 2]
```diff
@@ -943,9 +1026,12 @@ @string{ Tuci_E
 @string{ Tukey_PA             = "Tukey, Paul A." }
 @string{ Turgut_AE            = "Turgut, Ali Emre" }
 @string{ Turner_R             = "Turner, Ryan" }
+@string{ Tyrell_A             = "Tyrrell, Andy" }
 @string{ Tzoumas_G            = "Tzoumas, Georgios" }
 
 @string{ UbedaArriaza_K       = "Ubeda Arriaza, Keneth" }
+@string{ Ugur_E               = "Ugur, Emre" }
+@string{ Ulusoy_I             = "Ulusoy,{\.I}lkay" }
```
Comment (dagarzonr) [inline author.bib:1034]:
```suggestion
@string{ Ulusoy_I             = "Ulusoy, {\.I}lkay" }
```

File: author.bib [hunk 3]
```diff
@@ -978,11 +1065,15 @@ @string{ Viroli_M
 @string{ Vitanza_A            = "Vitanza, Alessandra" }
 @string{ Volz_RA              = "Volz, Richard A." }
 @string{ VonMammen_S          = "von Mammen, Sebastian" }
+@string{ Voyles_R             = "Voyles, Richard" }
 
+@string{ Vygen_J              = "Vygen, Jens" }
```
Comment (dagarzonr) [inline author.bib:1070]: Should be moved to the end of V, instead of the beginning of  W

File: author.bib [hunk 4]
```diff
@@ -1008,17 +1100,21 @@ @string{ Wilson_S
...
 @string{ Wischmann_S          = "Wischmann, Steffen" }
+@string{ Witwicki_W           = "Witwicki, Wojciech" }
 @string{ Wolf_DF              = "Wolf, Denis F." }
 @string{ Wolpert_D            = "Wolpert, David" }
 @string{ Wonham_WM            = "Wonham, W. M." }
-@string{ Wood_R               = "Wood, Robert" }
+@string{ Wood_RJ              = "Wood, Robert J." }
 @string{ Worn_H               = "Wörn, Heinz" }
 
+@string{ Wronski_P            = "Wroński, Paweł" }
```
Comment (dagarzonr) [inline author.bib:1110]: Should be moved to the end of W, instead of the beginning of X

File: proceedings.bib [hunk 5]
```diff
@@ -120,9 +124,11 @@ @string{ iros_2007
 @string{ iros_2012      = "2012 IEEE/RSJ International Conference On Intelligent Robots And Systems (IROS)" }
 @string{ iros_2013      = "2013 IEEE/RSJ International Conference On Intelligent Robots And Systems (IROS)" }
 @string{ iros_2018      = "2018 IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS)" }
+@string{ iscis_2007     = "2007 22nd International symposium on computer and information sciences (ISCIS)" }
```
Comment (dagarzonr) [inline proceedings.bib:127]:
```suggestion
@string{ iscis_2007     = "2007 22nd International Symposium on Computer and Information Sciences (ISCIS)" }
```

File: proceedings.bib [hunk 6]
```diff
@@ -134,6 +140,8 @@ @string{ ppsn_1998
 @string{ ppsn_2010      = "Parallel Problem Solving from Nature -- PPSN XI: 11th International Conference" }
 @string{ ppsn_2014      = "Parallel Problem Solving from Nature -- PPSN XIII: 13th International Conference" }
 
+@string{ robio_2010     = "2010 IEEE International Conference on Robotics and Biomimetics (ROBIO)" }
+@string{ robot_v1_2015  = "Robot 2015: Second Iberian Robotics Conference: Advances in Robotics, Volume 1" }
```
Comment (dagarzonr) [inline proceedings.bib:144]: I think the year should proceed the volume: robot_2015_v1

The volumes will belong to the proceedings of the conference of the same year and, therefore, the most important information is the year.

File: author.bib [hunk 7]
```diff
@@ -815,19 +886,22 @@ @string{ Rus_D
 @string{ Ryan_C               = "Ryan, Conor" }
 
 @string{ Saeedi_S             = "Saeedi, Sajad" }
+@string{ Sagoff_A             = "Sagoff, Amelia" }
 @string{ Saha_A               = "Saha, Arindam" }
-@string{ Sahin_E              = "\c{S}ahin, Erol" }
+@string{ Sahin_E              = "{\c S}ahin, Erol" }
```
Comment (dagarzonr) [inline author.bib:891]: The instructions at the beginning of this file state:

@comment{There is a known issue, if the first name starts with a non-ASCII character, the compilation for certain bibliography styles can break}

This PR escapes the first character in both first name and last name. Could you update it to:

@comment{There is a known issue, if the first or last name starts with a non-ASCII character, the compilation for certain bibliography styles can break}

File: bibliography.bib [hunk 8]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    pages        = {169--178},
+    publisher    = pub_SP,
+    address      = Berlin_DEU,
+    doi          = {10.1007/11553090_18},
+    note         = {},
+    annote       = {},
+}
+
+@InCollection {SchHam2011bioinspired,
+    title        = {{BEECLUST}: A swarm algorithm derived from honeybees},
```
Comment (dagarzonr) [inline bibliography.bib:7664]:
```suggestion
    title        = {{BEECLUST}: a swarm algorithm derived from honeybees},
```

File: bibliography.bib [hunk 9]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    number       = {6},
+    year         = {2005},
+    pages        = {1208--1214},
+    doi          = {10.1109/TRO.2005.853487},
+    note         = {},
+    annote       = {},
+}
+
+@Article {PayDaiEst-etal2001AR,
+    title        = {Pheromone Robotics},
```
Comment (dagarzonr) [inline bibliography.bib:7726]:
```suggestion
    title        = {Pheromone robotics},
```

File: bibliography.bib [hunk 10]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    pages        = {992--995},
+    doi          = {10.1038/35023164},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {SoaNavMar2015robot,
+    title        = {The {Khepera} {IV} mobile robot: performance evaluation, sensory data and software toolbox},
+    author       = Soares_JM # and # Navarro_I # and # Martinoli_A,
+    booktitle    = robot_v1_2015,
```
Comment (dagarzonr) [inline bibliography.bib:7868]: To be revised with respect to comment in book title

File: bibliography.bib [hunk 11]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    pages        = {37--42},
+    publisher    = pub_IEEE,
+    address      = Piscataway_NJ_USA,
+    doi          = {10.1109/MMAR.2017.8046794},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {PreiHonSukAya2017icra,
+    title        = {Crazyswarm: A large nano-quadcopter swarm},
```
Comment (dagarzonr) [inline bibliography.bib:7935]:
```suggestion
    title        = {Crazyswarm: a large nano-quadcopter swarm},
```

File: bibliography.bib [hunk 12]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    number       = {2},
+    year         = {1972},
+    pages        = {248--264},
+    doi          = {10.1145/321694.321699},
+    note         = {},
+    annote       = {},
+}
+
+@Article {LinKer1973OPRE,
+    title        = {An Effective Heuristic Algorithm for the Traveling-Salesman Problem},
```
Comment (dagarzonr) [inline bibliography.bib:8008]:
```suggestion
    title        = {An effective heuristic algorithm for the traveling-salesman problem},
```

File: bibliography.bib [hunk 13]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    pages        = {141--151},
+    publisher    = pub_ACM,
+    address      = NewYork_NY_USA,
+    doi          = {10.1145/87252.88081},
+    note         = {},
+    annote       = {},
+}
+
+@TechReport {LegGarKuc-etal2022techrep,
+    title        = {Tycho: a robust, {ROS}-based tracking system for robot swarms},
```
Comment (dagarzonr) [inline bibliography.bib:8053]:
```suggestion
    title        = {{Tycho}: a robust, {ROS}-based tracking system for robot swarms},
```

Comment (dagarzonr) [reply bibliography.bib:8053]: In this PR, other proper names are not protected (e.g., Crazyfly, Crazyswarm, etc). Please revise.

File: bibliography.bib [hunk 14]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    pages        = {45--52},
+    publisher    = pub_IFAAMAS,
+    address      = Richland_SC_USA,
+    doi          = {},
+    note         = {},
+    annote       = {},
+}
+
+@Article {ValFerHamDor2016AAMAS,
+    title        = {Collective decision with 100 kilobots: speed versus accuracy in binary discrimination problems},
```
Comment (dagarzonr) [inline bibliography.bib:8126]:
```suggestion
    title        = {Collective decision with 100 {Kilobots}: speed versus accuracy in binary discrimination problems},
```

File: bibliography.bib [hunk 15]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+    year         = {2011},
+    pages        = {908--917},
+    doi          = {10.1016/j.mechatronics.2010.08.003},
+    note         = {},
+    annote       = {},
+}
+
+@Article {Wal1950SCIAM,
+    title        = {An imitation of life},
+    author       = {W. Grey Walter},
```
Comment (dagarzonr) [inline bibliography.bib:8199]: To be added in author.bib

File: bibliography.bib [hunk 16]
```diff
@@ -7642,3 +7642,585 @@ @Article {PacBedCha-etal2019AL
...
+@InProceedings {ChiKimStaKim2021humanoids,
+    title        = {The {MIT} humanoid robot: design, motion planning, and control for acrobatic behaviors},
+    author       = Chignoli_M # and # Kim_D # and # StangerJones_E # and # Kim_S,
+    booktitle    = humanoids_2021,
+    editor       = Asfour_T,
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2021},
+    pages        = {1 -- 8},
```
Comment (dagarzonr) [inline bibliography.bib:8220]:
```suggestion
    pages        = {1--8},
```

**Review by dagarzonr — CHANGES_REQUESTED** [review]
Added few suggestions and comments.

File: author.bib (hunk 7 above)
Comment (jonaskuResearch) [reply author.bib:891]: I updated the header accordingly

**Review by dagarzonr — APPROVED** [review]

**Pull request merged**